README.md

```diff
@@ -71,3 +71,7 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets.
+
+There is a nice project at [bittner/lego-mindstorms-ev3-comparison](https://github.com/bittner/lego-mindstorms-ev3-comparison) highlighting differences LEGO MINDSTORMS EV3 Home Edition (#31313) and LEGO MINDSTORMS Education EV3 (#45544).
```

Comment:
... also referencing the robot programs from this repository and a few others.

(Could maybe be added?)

Reply:
Done in 41c6f42